hadoop-mapreduce/pom.xml

```diff
@@ -84,6 +84,13 @@
   </dependencies>
   <build>
     <plugins>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <configuration>
+          <omitVisitors>SharedVariableAtomicityDetector,ConstructorThrow</omitVisitors>
```

Comment:
In addition to previous comments about consolidating spotbugs config rules, this config here makes me wonder what rules are associated with "SharedVariableAtomicityDetector" and "ConstructorThrow", and why we're suppressing them here. It seems that suppression here is just because they are noisy?